Cargo.toml

```
@@ -64,6 +64,9 @@ walkdir = "2.5.0"
 [target.'cfg(target_os = "linux")'.dependencies]
 networkmanager = { package = "passcod-networkmanager", version = "=0.7.0-pre.1", optional = true }
 
+[target.'cfg(target_os = "windows")'.dependencies]
+windows = { version = "0.54.0", features = ["Win32_System_Console"] }
```

```suggestion
windows = { version = "0.54.0", features = ["Win32_System_Console"], optional = true }
```

and then add `dep:windows` to the `tamanu` feature.